source/OutfitInfoDisplay.cpp

```diff
@@ -471,6 +471,22 @@ void OutfitInfoDisplay::UpdateAttributes(const Outfit &outfit)
 	attributeValues.emplace_back(Format::Number(outfit.Range()));
 	attributesHeight += 20;
 
+	// Identify the dropoff at range and inform the player.
+	double fullDropoff = outfit.MaxDropoff();
+
+	if(fullDropoff != 1.)
+	{
+		attributeLabels.emplace_back("dropoff modifier:");
+		attributeValues.emplace_back(Format::Number(100 * fullDropoff) + "%");
+		attributesHeight += 20;
+		// Identify the ranges between which the dropoff takes place.
+		attributeLabels.emplace_back("dropoff range:");
+		attributeValues.emplace_back(Format::Number(outfit.DropoffStart())
+			+ " - " + Format::Number(outfit.DropoffEnd()));
```

Comment:
Are we missing an `attributesHeight += 20;` here?

Reply:
Good catch.